### ISSUE TITLE
AttributeError: module 'collections' has no attribute 'MutableMapping'

### ISSUE THREAD

**Issue by velle** [torfsen/barely_json#10]
I'm running Python 3.10.6 on Ubuntu, and trying out barely_json for the first time. I just installed barely_json-0.1.1 with pip. Importing barely_json gives the following error:

```
Traceback (most recent call last):
  File "/home/velle/a/parkscraper/cmdGetJsonToQueue.py", line 1, in <module>
    import barely_json
  File "/home/jdoe/.virtualenvs/testenv/lib/python3.10/site-packages/barely_json/__init__.py", line 15, in <module>
    from pyparsing import *
  File "/home/jdoe/.virtualenvs/testenv/lib/python3.10/site-packages/pyparsing.py", line 943, in <module>
    collections.MutableMapping.register(ParseResults)
AttributeError: module 'collections' has no attribute 'MutableMapping'
```

**Comment by torfsen** [comment]
Fixed by #11.

**Comment by torfsen** [comment]
Released in `1.0.0`.
